Boltdir/site-modules/gplt/tasks/create_control_repo_from_production_env.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+ENVIRONMENT=production
+PUPPET_PROD_ENV_DIR=/etc/puppetlabs/code/environments/$ENVIRONMENT
+BASEDIR=/opt/puppet
+CONTROL_REPO_URL=$BASEDIR/control-repo.git
+CHECKOUT_DIR=/root/control-repo
+
+mkdir -p $BASEDIR
+
+if [ -d $CONTROL_REPO_URL ]; then
+    echo "ERROR: The control repo destination already exists" >&2
+    exit 1
+fi
+
+git init --bare $CONTROL_REPO_URL
+git clone $CONTROL_REPO_URL $CHECKOUT_DIR
+cp -r $PUPPET_PROD_ENV_DIR/* $CHECKOUT_DIR
+rm -fr $CHECKOUT_DIR/modules
+cd $CHECKOUT_DIR \
+    || { echo "ERROR: Failed to cd into $CHECKOUT_DIR" >&2 ; exit 1 ; }
+git add .
+git commit -m "Initial commit"
+git branch -m $ENVIRONMENT
+git push -u origin $ENVIRONMENT
+cd $CONTROL_REPO_URL
+git symbolic-ref HEAD refs/heads/$ENVIRONMENT
```

Comment:
Should add output on where the repo was created, and where the checked out copy can be found.  It's at the top of the output, but the git output gets long, so having it again at the bottom will make it be easier to find and be seen.

Reply:
I have pushed an update incorporating a required parameter for the control repo path.  This value is returned as structured data to bolt at the end of the task.